packages/force_update/backend/Class1.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace backend;

```

Comment:
Are those files supposed to be here? I think with the current configuration, the backend will be included as part of the Flutter package

Reply:
good question. I think it should be in repo, but definitely should not be in package. I'll think whether to remove it completely or just move the backend folder higher to make sure it's not included in the package

Reply:
moved backend folder up